### DISCUSSION TITLE
Is it possible to pass module configs to server routes?

### DISCUSSION THREAD

**Discussion by brendonmatos** [nuxt/nuxt#22914] [Questions] [answered]
There is any way to achieve that?

**Comment by logotip4ik** [comment, marked as answer]
You can achieve that via virtual files. Example from elk client:

- [Definition of the virtual file with some config](https://github.com/elk-zone/elk/blob/5ef2996b4019d9279a7066b758a9247720765010/nuxt.config.ts#L175)
- [How to import from a virtual file](https://github.com/elk-zone/elk/blob/5ef2996b4019d9279a7066b758a9247720765010/server/utils/shared.ts#L13)
